app/routes/__private/user/employees.new.tsx

```diff
@@ -28,9 +36,14 @@ export const action: ActionFunction = (args) => {
         requestId,
         data,
         userId,
-      }).then(({ employeeUuid }) =>
-        redirect(`/user/employees/${employeeUuid}`)
-      ),
+      })
+        .then(({ employeeUuid }) => redirect(`/user/employees/${employeeUuid}`))
+        .catch((error) => {
+          console.error("Failed to create employee:", error);
```

Comment:
let's always email us instead of console.error, using `emailError`. That function should console.error in dev environment